src/TelnyxTests/Services/Wireless/WDRReports/WDRReportsServiceTest.cs

```diff
@@ -0,0 +1,102 @@
+﻿using System.Threading.Tasks;
+using Telnyx;
+using Telnyx.net.Entities.Wireless.WDRReports;
+using Telnyx.net.Services.Wireless.WDRReports;
+using Xunit;
+
+namespace TelnyxTests.Services.Wireless.MobileOperatorNetworks
+{
+    public class WDRReportsServiceTest : BaseTelnyxTest
+    {
+        private readonly WDRReportsService service;
+        private readonly ListWDRReportsOptions listOptions;
+        private readonly CreateWDRReportsOptions createOptions;
+        private readonly RequestOptions requestOptions;
+
+        private const string Id = "6a09cdc3-8948-47f0-aa62-74ac943d6c58";
+
+        public WDRReportsServiceTest(MockHttpClientFixture mockHttpClientFixture)
+            : base(mockHttpClientFixture)
+        {
+            this.service = new WDRReportsService();
+
+            this.listOptions = new ListWDRReportsOptions
+            {
+                PageNumber = 1,
```

Comment:
I believe the telnyx mock doesnt support these params to be passed, so just instantiate a blank ListOptions object

Reply:
changed to blank listoptions